src/tokens/transformation/android/formats/fontStyle.js

```diff
@@ -11,20 +11,14 @@ const printDescription = (description) =>
 
 module.exports = ({ dictionary, platform, options = {}, file }) => {
   const fontStyles = dictionary.allTokens
-    // remove underlined
-    .filter(
-      (compositeToken) =>
-        compositeToken.original.value.textDecoration !== 'underline'
-    )
+    .filter((compositeToken) => compositeToken.type === 'custom-fontStyle')
     // create style
     .map((compositeToken) => {
       return (
-        `  <style name="${camelCase(compositeToken.name)}">\n` +
+        `  <style name="${changeCase.pascalCase(compositeToken.name.replace("font_android_", ""))}">\n` +
```

Comment:
Should this one also be `snake_case`?

Reply:
styles in chromium android are defined in pascalCase so  i kept it that way. 
for ref : https://source.chromium.org/chromium/chromium/src/+/main:chrome/android/java/res/values/styles.xml;l=256?q=styles.xml&ss=chromium